src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/MoveServiceTest.kt

```diff
@@ -1,15 +1,9 @@
 package uk.gov.justice.digital.hmpps.pecs.jpc.service
 
-import com.nhaarman.mockitokotlin2.eq
+import com.nhaarman.mockitokotlin2.mock
```

Comment:
There was a reasonable amount of refactoring in this test class (mainly removal of unnecessary code).